__tests__/app.test.js

```diff
@@ -127,17 +118,6 @@ describe("/api/articles/:article_id", () => {
           expect(res.body.msg).toBe("bad request");
         });
     });
-    test("404 - given correct id with no data, returns message: not found", () => {
```

Comment:
Would also say this is a valid test seeing as you have to consciously put in a promise reject in the model to make this happen